qcfractal/interface/collections/torsiondrive_dataset.py

```diff
@@ -141,14 +142,18 @@ def add_entry(self,
             raise KeyError(f"Record {name} already in the dataset.")
 
         self.data.records[lname] = record
+        self.save()
```

Comment:
Same question about `save` here in a `get` method

Reply:
This is an `add_entry` we must call save here to update the record. Slower, but safer version of Dataset `add_entry`, we do not expect tens of thousands of entries here.